From a2c66927b75547cee1db1340a67449ded0812df3 Mon Sep 17 00:00:00 2001
From: Li xin <lixin.fnst@cn.fujitsu.com>
Date: Wed, 19 Aug 2015 11:13:46 +0900
Subject: [PATCH] ribcl.py.in: Warning Fix

WARNING: QA Issue: /usr/lib/stonith/plugins/stonith2/ribcl.py_cluster-glue-plugin-stonith2-ribcl
contained in package cluster-glue-plugin-stonith2-ribcl requires /usr/local/bin/python,
but no providers found in its RDEPENDS [file-rdeps]

Add target python handling
Allow the build to specify a path for the python instead of reusing
the PYTHON variable which can lead to inconsistencies if we are cross
compiling.

Upstream-Status: pending

Signed-off-by: Li Xin <lixin.fnst@cn.fujitsu.com>
---
 lib/plugins/stonith/ribcl.py.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/plugins/stonith/ribcl.py.in b/lib/plugins/stonith/ribcl.py.in
index 14e070c..01205e3 100644
--- a/lib/plugins/stonith/ribcl.py.in
+++ b/lib/plugins/stonith/ribcl.py.in
@@ -1,4 +1,4 @@
-#!@PYTHON@
+#!@TRAGET_PYTHON@
 
 
 #
-- 
1.8.4.2

